docs/docs/cli/reference.md

```diff
@@ -54,6 +54,19 @@ airy api signup [flags]
       --cli-config string   config file (default is $HOME/.airy/cli.yaml)
 ```
 
+***
+## Cheat Sheet
+
+| Command             | Effect                                                                             |
+| ------------------- | ---------------------------------------------------------------------------------- |
+| `airy init`         | Inits your airy configuration                                                      |
+| `airy status`       | Reports the status of an Airy Core instance                                        |
+| `airy config apply` | Applies configuration values from airy.yaml configuration to an Airy Core instance |
+| `airy ui`           | Opens the Airy Core UI in your local browser                                       |
+| `airy api signup`   | Creates a default user                                                             |
+| `airy api login`    | Returns a valid jwt token                                                          |
+| `airy api login`    | Returns a valid jwt token                                                          |
+| `airy --help`       | Prints all available commands with parameters                                      |
```

Comment:
the file is auto generated so this will break the build.

But the actual plan for this issue was to remove the cheat sheet all together (I know it was not clear from the issue... but I "forgot" to comment there... and I assumed I'd do this myself :)) so fortunately the fix is pretty simple. We keep all the other changes you make (because they're good!) and revert changes to this file.